google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -243,6 +287,252 @@ void DeleteCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
+      location, 3, google::cloud::bigtable::ClusterConfig::HDD);
+  auto cluster =
+      instance_admin.CreateCluster(cluster_config, instance_id, cluster_id_2);
+  std::cout << "Cluster Created " << cluster.get().name() << std::endl;
+  // [END bigtable_create_cluster]
+
+  // [START bigtable_delete_cluster]
+  // Delete Cluster
+  std::cout << std::endl << "Deleting Cluster: ";
+  instance_admin.DeleteCluster(instance_id, cluster_id_2);
+  std::cout << " Done" << std::endl;
+  // [END bigtable_delete_cluster]
+
+  // [START bigtable_delete_instance]
+  // Delete Instance
+  std::cout << std::endl << "Deleting Instance: ";
+  instance_admin.DeleteInstance(instance_id.get());
+  std::cout << " Done" << std::endl;
+  // [END bigtable_delete_instance]
+}
+
+// This full example demonstrate various instance operations
+// by initially creating development instance
+void RunFullExample2(std::string project_id, int argc, char* argv[]) {
+  if (argc != 4) {
+    throw Usage{
+        "run-full-example-1: <project-id> <instance-id> <cluster-id> <zone>"};
+  }
+  google::cloud::bigtable::InstanceId instance_id(ConsumeArg(argc, argv));
+  google::cloud::bigtable::ClusterId cluster_id(ConsumeArg(argc, argv));
+  std::string const zone = ConsumeArg(argc, argv);
+
+  // Create an instance admin endpoint.
+  google::cloud::bigtable::InstanceAdmin instance_admin(
+      google::cloud::bigtable::CreateDefaultInstanceAdminClient(
+          project_id, google::cloud::bigtable::ClientOptions()));
+
+  std::cout << std::endl << "Check Instance exists: " << std::endl;
+  auto instances = instance_admin.ListInstances();
+  auto instance_name =
+      instance_admin.project_name() + "/instances/" + instance_id.get();
+  bool instanceExists =
+      instances.end() !=
+      std::find_if(
+          instances.begin(), instances.end(),
+          [&instance_name](google::bigtable::admin::v2::Instance const& i) {
+            return i.name() == instance_name;
+          });
+  // Create instance if does not exists
+  if (not instanceExists) {
+    // [START bigtable_create_dev_instance]
+    std::cout << std::endl << "Creating a DEVELOPMENT Instance: ";
+    // Create development instance
+    google::cloud::bigtable::DisplayName display_name("Put description here");
+
+    auto cluster_config = google::cloud::bigtable::ClusterConfig(
+        zone, 0, google::cloud::bigtable::ClusterConfig::HDD);
+    google::cloud::bigtable::InstanceConfig config(
+        google::cloud::bigtable::InstanceId(instance_id), display_name,
+        {{cluster_id.get(), cluster_config}});
+    config.set_type(google::cloud::bigtable::InstanceConfig::DEVELOPMENT);
+
+    auto future = instance_admin.CreateInstance(config).get();
+    std::cout << " Done" << std::endl;
+    // [END bigtable_create_dev_instance]
+  } else {
+    std::cout << std::endl
+              << "Instance " << instance_id.get() << " already exists."
+              << std::endl;
+    return;
+  }
+
+  // List Instances
```

Comment:
Same comment about the redundant comments + messages.

Reply:
Removed.